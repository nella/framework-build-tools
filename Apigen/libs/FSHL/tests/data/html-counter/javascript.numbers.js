var test = -1.5 - 0.5 + -.5;