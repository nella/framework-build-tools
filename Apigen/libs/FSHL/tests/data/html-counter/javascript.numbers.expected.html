<html><head><meta http-equiv="content-type" content="text/html; charset=utf-8" /><link rel="stylesheet" type="text/css" href="../../../style.css" media="all" /></head><body><pre>
<span class="line">1: </span><span class="js-keywords1">var</span><span class="js-out"> test = -</span><span class="js-num">1.5</span><span class="js-out"> - </span><span class="js-num">0.5</span><span class="js-out"> + -</span><span class="js-num">.5</span><span class="js-out">;</span>
</pre></body></html>